django_toolkit/middlewares.py

```diff
@@ -11,3 +17,27 @@ class VersionHeaderMiddleware(object):
     def process_response(self, request, response):
         response['X-API-Version'] = settings.TOOLKIT['API_VERSION']
         return response
+
+
+class AccessLogMiddleware(object):
+
+    LOG_FORMAT = (
+        u'[{app_name}] '
+        u'{response.status_code} {request.method} {request.path}'
+    )
+
+    UNKNOWN_APP_NAME = 'unknown'
+
+    def process_response(self, request, response):
+        app = get_oauth2_app(request)
+        app_name = self.UNKNOWN_APP_NAME
+
+        if app:
+            app_name = app.name
```

Comment:
maybe here you could use
`app_name = getattr(app, 'name', self. UNKNOWN_APP_NAME)`